toffy/bin_extraction.py

```diff
@@ -0,0 +1,34 @@
+from toffy.json_utils import list_moly_fovs
+from mibi_bin_tools import bin_files, io_utils
+
+
+def extract_missing_fovs(bin_file_dir, extraction_dir, panel, extract_intensities, replace=True):
+    """Check for already extracted FOV bin files, and extract the remaining (excluding moly)
+
+    Args:
+        bin_file_dir (str): path to directory containing the bin and json files
+        extraction_dir (str): path to directory of already extracted FOVs
+        panel (pd.DataFrame): file defining the panel info for bin file extraction
+        extract_intensities (bool): whether to extract intensities from the bin files
+        replace (bool): whether to replace pulse images with intensity
+    """
+
+    # retrieve all fov names from base_dir and extracted fovs from extraction_dir
+    fovs = io_utils.remove_file_extensions(io_utils.list_files(bin_file_dir, substrs='.bin'))
```

Comment:
Does this work yet?

Reply:
yeah I used `mini_bin_tools.io_utils()` here since we haven't re-released ark yet